src/marqo/core/structured_vespa_index/structured_vespa_schema.py

```diff
@@ -275,10 +276,10 @@ def _generate_rank_profiles(self, marqo_index: StructuredMarqoIndex) -> List[str
             rank_profiles.append(f'query({field.embeddings_field_name}): 0')
         rank_profiles.append(f'query({common.QUERY_INPUT_EMBEDDING}) tensor<float>(x[{model_dim}])')
         rank_profiles.append(f'query({common.QUERY_INPUT_BM25_AGGREGATOR}): 0')
-        rank_profiles.append(f'query({common.QUERY_INPUT_SCORE_MODIFIERS_MULT_WEIGHTS_LEXICAL}) tensor<double>(p{{}})')
-        rank_profiles.append(f'query({common.QUERY_INPUT_SCORE_MODIFIERS_ADD_WEIGHTS_LEXICAL}) tensor<double>(p{{}})')
-        rank_profiles.append(f'query({common.QUERY_INPUT_SCORE_MODIFIERS_MULT_WEIGHTS_TENSOR}) tensor<double>(p{{}})')
-        rank_profiles.append(f'query({common.QUERY_INPUT_SCORE_MODIFIERS_ADD_WEIGHTS_TENSOR}) tensor<double>(p{{}})')
+        rank_profiles.append(f'query({constants.QUERY_INPUT_SCORE_MODIFIERS_MULT_WEIGHTS_LEXICAL}) tensor<double>(p{{}})')
```

Comment:
@farshidz Mentioned that it's better to use `textwrap.dedent` for unstructured, since indexes will have the same rank profiles, but for structured, we use single line `append` statements to customize whether lexical/tensor/hybrid rank profiles are added.